pages/index.tsx

```diff
@@ -64,6 +79,24 @@ const HomePage = () => {
     await fetchTimeEntries();
   };
 
+  const filterTimeEntries = (timeEntryList: TimeEntryInterface[]) => {
+    if (clientFilter === "") {
+      setFilteredTimeEntries(timeEntryList);
+      return;
+    }
+
+    setFilteredTimeEntries(timeEntryList.filter((timeEntry) => timeEntry.client === clientFilter));
```

Comment:
As I told, you know it is also a possibility to filter when the data is returned instead of yet another Store item. (keep this as necessary as possible)

Reply:
No need to change for now  😄